dashboard/db/migrate/20160502025309_add_kind_to_survey_results.rb

```diff
@@ -0,0 +1,15 @@
+class AddKindToSurveyResults < ActiveRecord::Migration
+  def change
+    # The `kind` column will be used to store the kind of survey, e.g.,
+    # DiversityYYYY, NetPromoterScoreYYYY, etc.
+    add_column :survey_results, :kind, :string, :after => :user_id
+    add_index :survey_results, :kind
+
+    reversible do |direction|
+      direction.up do
+        # On 2016-04-30, existing rows are the diversity survey from March 2016.
+        SurveyResult.update_all(kind: 'Diversity2016')
+      end
+    end 
+  end
+end
```

Comment:
Given that `created_at == updated_at` for all existing rows, I don't think I'm concerned with whether or not `updated_at` is updated.